velox/dwio/parquet/reader/CMakeLists.txt

```diff
@@ -30,14 +30,12 @@ velox_add_library(
 
 velox_link_libraries(
   velox_dwio_native_parquet_reader
-  velox_dwio_parquet_thrift
```

Comment:
We don't need this because `velox_dwio_parquet_common` already depends on `velox_dwio_parquet_thrift`.